exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,7 @@
 const name = " Daniel  ";
 
-console.log(message);
+let message = "My name is " + firstName + " and my name is 5 characters long";
```

Comment:
1. Variable `firstName` doesn't exist here. You need to use `name` instead.
2. Use a string method to calculate the length of your name rather than writing it directly here as `5`. 
3. You need to use `.trim()` on the name to clear extra spacing around it.
4. It's almost always better to use this format for string concatinations:
Like this:
```
name = name.trim()
let message =`My name is ${name} and my name is ${name.length} characters long`
```